app/components/no-account.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+});
```

Comment:
I think you could safely delete this file; like with a lot of things in Ember (routes, controllers, for instance), a basic implementation will be created if one isn’t given. I think the default would be this exact file!